config/routes.rb

```diff
@@ -11,7 +11,12 @@
   resources :technologies, only: [:create]
   resources :favourite_technologies, only: [:create, :destroy]
 
-  resources :recipes, except: [:destroy]
+  resources :recipes, except: [:destroy] do
+    scope module: :recipes do
```

Comment:
This is a little trick to convey in the folder structure of your app that the ingredients belong to recipes and that the controller meant to be used is inside a folder:
```text
app/
  controllers/
    recipes/
      ingredients_controller.rb
  views/
    recipes/
      ingredients/
        destroy.turbo_stream.erb
```